src/workflows/airqo_etl_utils/data_api.py

```
@@ -1,15 +1,18 @@
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)